strapi/strapi
# Select and run codemods only

### What does it do?
It adds a new command to execute a combination of selected codemods instead of running all of them.

### Why is it needed?

To run selected codemods

### How to test it?

Execute the upgrade command with the `codemods` option and you should be able to select the different codemods available to run.

### Related issue(s)/PR(s)
